src/services/gist.js

```diff
@@ -0,0 +1,16 @@
+export const retrieveAllSchedules = async (url) => {
+  let references = [];
+
+  try {
```

Comment:
Is this really necessary? Can't those exceptions be handled outside of the function's scope?